lib/jenkins_pipeline_builder/extension_dsl.rb

```diff
@@ -17,6 +17,6 @@ def job_attribute(&block)
 
   JenkinsPipelineBuilder.registry.register([:job], set)
   versions = set.extensions.map(&:min_version)
-  puts "Successfully registered #{set.name} for versions #{versions}" if set.announced
+  JenkinsPipelineBuilder.logger.info "Successfully registered #{set.name} for versions #{versions}" if set.announced
```

Comment:
Since these are defined on Main, removing that makes it dependent on where the method is executed. I think it is safer to explicitly call it on the `JenkinsPipelineBuilder` object. I should probably change the above as well.